src/pages/lessons/[slug].tsx

```diff
@@ -109,6 +112,13 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
 
   if (!lesson) return null
 
+  React.useEffect(() => {
```

Comment:
if it isn't in a useEffect it cries in the console.

Reply:
I need to dive into the react/xstate/hooks demos